cfgov/jinja2/v1/_includes/organisms/post-preview.html

```diff
@@ -164,15 +163,15 @@ <h3 class="o-post-preview_title">
             <div class="o-post-preview_byline-group">
             {% for author in post.get_authors() %}
                 {% if loop.index == 1  %}
-                    By <a href="{{ page_url }}?authors={{ author.slug }}">
+                    By <a href="{{ url }}?authors={{ author.slug }}">
```

Comment:
It's not obvious from the diff, but the changes here mean that `url` is either an empty string, or is passed in by the calling template.  The only time it is passed in is from a page with a post preview snapshot on it, of which there are two (http://localhost:8000/policy-compliance/amicus/, http://localhost:8000/policy-compliance/notice-opportunities-comment/), and otherwise we want a relative URL here, rather than hardcording the page, which presents issues for caching.